jbmc/regression/jbmc/string_literal1/test.desc

```diff
@@ -1,6 +1,6 @@
-KNOWNBUG
-array1.class
-
+CORE
+string_literal1.class
+--cp `../../../../scripts/format_classpath.sh . ../../../src/java_bytecode/library/core-models.jar`
```

Comment:
I think this jar file should be used instead `../../../lib/java-models-library/target/core-models.jar`